internetarchive/iaux-popup
# Popup Component

New component: hover popup on tiles
Can be click-only or hover
![ezgif com-gif-maker (7)](https://user-images.githubusercontent.com/7840857/114083021-5890c900-9863-11eb-910a-4f5ec31d6572.gif)




To Test: 
- go to `https://www-isa.archive.org/search.php?query=beyonce&sin=TXT` 
  - scroll down results to see result file name inside of drawer, should fade out at end of line
  - hover over, popup shows
- clone repo -> `yarn install && yarn start` -> go to demo page to see hover example
